### DISCUSSION TITLE
Configurable menu item Name

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [music-assistant/support#515] [Feature requests and ideas]
Please consider allowing a user configurable menu item name in the left menu.

just like we can rename the default HACS to Resources ( or anything else), I’d love so call it Music .

thanks for considering

![74B3D93B-40FE-49C9-9C0C-7824A25C50F3](https://user-images.githubusercontent.com/33354141/175648016-abea66d0-6631-459d-a46c-561ed3770bfd.jpeg)

**Comment by marcelveldt** [comment]
That is already possible. Just click "rename" on the integration's card:

<img width="957" alt="Schermafbeelding 2022-06-25 om 01 00 55" src="https://user-images.githubusercontent.com/6389780/175720984-7e3d88cc-5d7b-4a7d-bc8d-27a212028e92.png">

**Reply by Mariusthvdb** [reply]
My bad, completely missed that…
Sorry

<img width="228" alt="Schermafbeelding 2022-06-25 om 11 05 41" src="https://user-images.githubusercontent.com/33354141/175766497-9d89df22-7229-41fb-a6c1-0249f42b5280.png">

Nice!

On the minute details....... guess its too much to ask to consider another icon. For me, it would be reflecting the integration more if we could see a music note in the icon (see the slimprotoe player in your screenshot) , and MDI has several of them that are suitable imho.

Would make it standout more compared to the Media icon as well 

no dealbreaker, just a thought

**Comment by OzGav** [comment]
Relates to v1
